packages/components/dialog/src/dialog.ts

```diff
@@ -81,6 +56,11 @@ export const dialogProps = buildProps({
     type: Number,
   },
 } as const)
+
+export const dialogContentEmits = {
+  close: () => true,
+}
+
 export type DialogProps = ExtractPropTypes<typeof dialogProps>
```

Comment:
export type DialogContentEmits

Reply:
I don't think this type is necessary since it is only internally used.

Reply:
Just be consistent